inspire/base/static/less/base/variables.less

```diff
@@ -21,6 +21,12 @@
 
 @import (reference) "../inspire.less";
 
+// New design colors
+@blue: #2C3E50;
```

Comment:
As here you are touching the same file as in https://github.com/inspirehep/inspire-next/pull/418 it would be better if you create a single PR with both commits so that you can fix the conflicts.